app/renderer/actions/Session.js

```diff
@@ -338,6 +342,7 @@ export function changeServerType (serverType) {
 export function setServerParam (name, value) {
   return (dispatch, getState) => {
     dispatch({type: SET_SERVER_PARAM, serverType: getState().session.serverType, name, value});
+    getRunningSessions()(dispatch, getState);
```

Comment:
Just made the changes now

* It uses a combobox instead of select so that you can hardcode values
* Debounced it 500ms so it doesn't do the `getRunningSessions` on every keystroke